Libplanet.Explorer/GraphTypes/ActionType.cs

```diff
@@ -13,13 +16,54 @@ public ActionType()
                 "Arguments",
                 resolve: ctx =>
                 {
+                    string ConvertKeyToString(IKey key)
+                    {
+                        switch (key)
+                        {
+                            case Text t:
+                                return t.Value;
+                            case Binary binary:
+                                return binary.Value.ToString();
```

Comment:
Doesn't `.ToString()` generate the same representation regardless of its content?

```csharp
> (new byte[] { 0x00, 0x01, 0x02 }).ToString()
"System.Byte[]"
> (new byte[] { 0x00, 0x00, 0x00 }).ToString()
"System.Byte[]"
```

Reply:
Oh, It's my mistake 😢